fedimint-server/src/net/peers.rs

```diff
@@ -70,9 +70,6 @@ struct PeerConnection<T> {
 pub struct NetworkConfig {
     /// Our federation member's identity
     pub identity: PeerId,
-    /// Our listen address for incoming connections from other federation
-    /// members
-    pub bind_addr: SocketAddr,
```

Comment:
Not sure why the bind address is removed here, afaik this struct isn't ever (de-)serialized? This just leads to the bind address becoming another argument in all the functions we are passing `NetowrkConfig` to as far as I can tell.

Reply:
Hmm... Had `Serialize`, so I assumed it's serialized. I'll investigate.